src/pages/commits/ActiveCommits/buildVariantCard/FailedTaskStatusIcon.tsx

```diff
@@ -0,0 +1,80 @@
+import { useLazyQuery } from "@apollo/client";
+import styled from "@emotion/styled";
+import IconButton from "@leafygreen-ui/icon-button";
+import Tooltip from "@leafygreen-ui/tooltip";
+import { Skeleton } from "antd";
+import { Link } from "react-router-dom";
+import { TaskStatusIcon } from "components/TaskStatusIcon";
+import { getTaskRoute } from "constants/routes";
+import {
+  GetFailedTaskStatusIconTooltipQuery,
+  GetFailedTaskStatusIconTooltipQueryVariables,
+} from "gql/generated/types";
+import { GET_FAILED_TASK_STATUS_ICON_TOOLTIP } from "gql/queries";
+import { msToDuration } from "utils/string";
+
+interface FailedStatusIconProps {
+  taskId: string;
+  status: string;
+}
+export const FailedTaskStatusIcon: React.FC<FailedStatusIconProps> = ({
+  taskId,
+  status,
+}) => {
+  const [loadData, { data, loading }] = useLazyQuery<
+    GetFailedTaskStatusIconTooltipQuery,
+    GetFailedTaskStatusIconTooltipQueryVariables
+  >(GET_FAILED_TASK_STATUS_ICON_TOOLTIP, { variables: { taskId } });
+  const { displayName, timeTaken } = data?.task ?? {};
+  const { testResults } = data?.taskTests ?? {};
+
+  return (
+    <Tooltip
+      usePortal={false}
+      align="top"
+      justify="middle"
+      popoverZIndex={1}
+      trigger={
+        <div
+          onMouseOver={() => loadData()}
+          onFocus={() => loadData()}
+          data-cy="failed-task-status-icon"
+        >
+          <Link
+            data-cy="task-status-icon"
+            to={getTaskRoute(taskId)}
+            key={`task_${taskId}`}
+          >
+            <IconButton aria-label="task icon">
+              <TaskStatusIcon status={status} size={16} />
+            </IconButton>
+          </Link>
+        </div>
+      }
+      triggerEvent="hover"
+    >
+      <div data-cy="failed-task-status-icon-tooltip">
+        {loading ? (
+          <Skeleton />
+        ) : (
+          !!data && (
+            <>
+              <TooltipTitle data-cy="failed-task-status-icon-tooltip-title">
+                {displayName} - {msToDuration(timeTaken)}
+              </TooltipTitle>
```

Comment:
We should try to differentiate this from the test results them selves.
We should try to make the task name visually distinctive from the test results. 
Are there designs for this? If not maybe just make the test name bold and we can discuss it in the next design review. 
![image](https://user-images.githubusercontent.com/4605522/137387660-ec38df6e-fb7a-478a-8d17-c238cf81c828.png)